lib/runtime/GetChunkFilenameRuntimeModule.js

```diff
@@ -228,7 +228,7 @@ class GetChunkFilenameRuntimeModule extends RuntimeModule {
 		const url =
 			dynamicFilename &&
 			compilation.getPath(JSON.stringify(dynamicFilename), {
-				runtime: `" + "${chunk.runtime}" + "`,
+				runtime: `" + ${JSON.stringify(c.runtime)} + "`,
```

Comment:
@sokra , A little mistake here. In this line "c" - is not a chunk. Variable name here - "chunk". I committed this fix

Reply:
Thanks. copy paste 😆